foundry/pixi/containers/canvasLayers/effectsLayer.d.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * A CanvasLayer for displaying visual effects like weather, transitions, flashes, or more
+ */
+declare class EffectsLayer extends CanvasLayer {
+  constructor();
+
+  /**
+   * The weather overlay container
+   * @defaultValue `null`
+   */
+  weather: PIXI.Container | null;
+
+  /**
+   * The currently active weather effect
+   * @defaultValue `null`
+   */
+  weatherEffect: SpecialEffect | null;
+
+  /**
+   * Track any active emitters within this Scene
+   * @defaultValue `[]`
+   */
+  emitters: unknown[]; // I don't believe that this is ever populated right now - Bolts
+
+  /** @override
+   * @defaultValue `mergeObject(super.layerOptions, { zIndex: 300 })`
+   */
```

Comment:
We should be consistent with our Multiline doc comments. They should leave the first line with the two asterisks blank.

Reply:
Hm. I thought that was something eslint/prettier would catch. I'll keep an eye out for that in the future.